locales/en/yourloops.json

```diff
@@ -132,12 +134,14 @@
   "delete": "Delete",
   "delete-medical-report": "Delete medical report",
   "delete-warning": "Are you sure you want to delete {{ fileName }}?",
+  "diabete-type": "Diabete type",
   "diagnosis": "Diagnosis",
   "dialog-pdf-options-custom-range": "or a customized period",
   "dialog-pdf-options-format": "Choose an output format",
   "dialog-pdf-options-output-format-csv": "CSV",
   "dialog-pdf-options-output-format-pdf": "PDF",
   "dialog-pdf-options-presets": "Choose a fixed period",
+  "double-dot": "{{label}}:",
```

Comment:
I'm not sure of the term "double dot", I think the appropriate name for this punctuation sign is "colon"?